UPDATING.md

```diff
@@ -0,0 +1,26 @@
+## Updating this example's dependencies
+
+When you feel like modifying any of this example's dependencies (Mbed Crypto,
+Cryptoauthlib), for example to update the secure element driver interface,
+you'll need to make your changes across a handful of different components.
+
+1. Update SE driver interface (and tests) in Mbed Crypto (`mbed-crypto`)
+1. Save the changes to a branch in your fork of Mbed Crypto (`mbed-crypto`)
+1. Update Mbed OS (`mbed-os`) with your Mbed Crypto fork's SE driver changes
+1. Save the changes to a branch in your fork of Mbed OS (`mbed-os`)
+    1. `cd features/mbedtls/mbed-crypto/importer`
+    1. Edit `CRYPTO_RELEASE` to point to your branch
```

Comment:
There's only the one file in this folder, but I think it might be worth being explicit that you're editing the Makefile here.

Reply:
Added reference to the Makefile